sdk/servicebus/service-bus/test/utils/testUtils.ts

```diff
@@ -328,8 +318,8 @@ export async function getSenderReceiverClients(
         )
       };
     }
-    case TestClientType.UnpartitionedQueueWithSessions: {
-      if (env[EnvVarKeys.CLEAN_NAMESPACE]) {
+    case TestClientType.UnpartitionedQueueWithSessions:
+      {
```

Comment:
This `{` got pushed to the next line by mistake I believe...

Reply:
Oops yes - fixed it - though the scoping for each of the case seems unnecessary.